.github/workflows/yapf_check.yml

```diff
@@ -0,0 +1,12 @@
+name: YAPF Formatting Check
+on: [push]
+jobs:
+  formatting-check:
```

Comment:
Why wasn't this added to the existing workflow? It could replace the current lint step that we don't care about anymore.

Reply:
good point- I originally couldn't figure out how to add to existing workflow but I figured it out